### ISSUE TITLE
Add in-app controls (not just via notification)

### ISSUE THREAD

**Issue by no-identd** [giraffine/Android-Project#18]
'Recent' Android versions block overlays from applying to the notification drawer — and in lieu of root and in view of the widespread existence of devices that, while having a 'dark' theme, still have all the notifications bright white, it'd be useful to have some actually app screen so one doesn't have to open the way way way too bright notification drawer when one wants to change the dimming level.

**Comment by no-identd** [comment]
@webhat any chance of this in your fork?

**Comment by webhat** [comment]
No, this is where I got stuck last time I worked on it. For security reasons Android rightly doesn't allow overlays on the draw, this requires a patch to AOSP.

**Comment by no-identd** [comment]
@webhat I know, that's not what I asked about here tho. What I meant is having an app screen that'd open upon tapping the app icon, to configure the dimming intensity /there/, to avoid having to open the hyper-bright notification drawer. (because currently, the app notification is the only place where one can adjust the dimming intensity)
